src/ruby/yast/builtins.rb

```diff
@@ -492,42 +490,42 @@ def self.toset array
 
     # Check if map has a certain key
     # @deprecated use {::Hash#haskey?}
-    def self.haskey map, key
+    def self.haskey(map, key)
       return nil if map.nil? || key.nil?
-      map.has_key? key
+      map.key? key
```

Comment:
Apparently Rubocop does not care about parens at the *call* point, only at the *definition* point.